src/DownloadVideoURL.ts

```diff
@@ -62,6 +62,11 @@ export class DownloadVideoURL {
                     }
                     reject(e);
                 });
+
+                this.#httpRequest.on('timeout', () => {
```

Comment:
So what would happen currently? SoftTimeLimit?

Reply:
I don't know. We don't have so many soft time limits, it's just an idea

Reply:
Are we sure this event gets emitted or is this just a way to find out @stepancar ?